docs/media/basic-compositions/Screenshot_20221216_100511.png:Zone.Identifier

```diff
@@ -0,0 +1,2 @@
+[ZoneTransfer]

```

Comment:
I think this is a WSL metadata file. Please remove it. I opened #3535 to prevent it in the future.